### PR TITLE
Missing error states

### PR DESCRIPTION
**Pull request by ismailToyran**
### Project organization
- Related [trello card](https://trello.com/c/YQkkcGsn/654-missing-error-when-accepting-bid-thats-no-longer-valid)

### Description
Adds missing error toasters to some components

### How to test
Main flow to be tested;
- Bid on any NFT
- Transfer you balance to your other account
- When seller accepts your bid, it should throw error as you don't have that balance anymore

### Checklist
- [x] Base branch of the PR is `dev`

### CODE REVIEW

File: pages/explore/collections.tsx [hunk 1]
```diff
@@ -102,11 +104,16 @@ const CollectionsPage: NextPage<Props> = ({}) => {
           pathname,
           query: { ...query, orderBy, page: undefined },
         })
+      } catch (e) {
+        toast({
+          title: formatError(e),
+          status: 'error',
+        })
```
Comment (ismailToyran) [inline pages/explore/collections.tsx:111]: Let me know if it doesn't make sense to show toaster changing order so I can revert

File: pages/notification.tsx [hunk 2]
```diff
@@ -77,10 +79,15 @@ const NotificationPage: NextPage<Props> = ({ currentAccount }) => {
         variables: { cursor: data?.notifications?.pageInfo.endCursor },
         updateQuery: concatToQuery('notifications'),
       })
+    } catch (e) {
+      toast({
+        title: formatError(e),
+        status: 'error',
+      })
```
Comment (ismailToyran) [inline pages/notification.tsx:86]: Let me know if it doesn't make sense to show toaster loading more notifications so I can revert

**Comment by ismailToyran** [comment]
These were the only ones that were without `try catch`

**Review by antho1404 — APPROVED** [review]
Tested to remove the allowance and at least we now have a message about it that is quite clear

Screenshot 2023-03-07 at 21 36 23

**Pull request merged**